packages/app-mobile/components/screens/ConfigScreen/ConfigScreen.tsx

```diff
@@ -530,33 +531,15 @@ class ConfigScreenComponent extends BaseScreenComponent<ConfigScreenProps, Confi
 		}
 
 		if (section.name === 'joplinCloud') {
-			const label = _('Email to note');
-			const description = _('Any email sent to this address will be converted into a note and added to your collection. The note will be saved into the Inbox notebook');
-			const isEmailToNoteAvailableInAccount = this.props.settings['sync.10.accountType'] !== 1;
-			const inboxEmailValue = isEmailToNoteAvailableInAccount ? this.props.settings['sync.10.inboxEmail'] : '-';
 			addSettingComponent(
-				<View key="joplinCloud">
-					<View style={this.styles().styleSheet.settingContainerNoBottomBorder}>
-						<Text style={this.styles().styleSheet.settingText}>{label}</Text>
-						<Text style={this.styles().styleSheet.settingTextEmphasis}>{inboxEmailValue}</Text>
-					</View>
-					{
-						!isEmailToNoteAvailableInAccount && (
-							<View style={this.styles().styleSheet.settingContainerNoBottomBorder}>
-								<Text style={this.styles().styleSheet.descriptionAlert}>{_('Your account doesn\'t have access to this feature')}</Text>
-							</View>
-						)
-					}
-					{
-						this.renderButton(
-							'sync.10.inboxEmail',
-							_('Copy to clipboard'),
-							() => isEmailToNoteAvailableInAccount && Clipboard.setString(this.props.settings['sync.10.inboxEmail']),
-							{ description, disabled: !isEmailToNoteAvailableInAccount },
-						)
-					}
-				</View>,
-				[label, description],
+				<JoplinCloudConfig
+					accountType={this.props.settings['sync.10.accountType']}
+					inboxEmail={this.props.settings['sync.10.inboxEmail']}
+					userEmail={this.props.settings['sync.10.userEmail']}
+					website={this.props.settings['sync.10.website']}
+					styles={this.styles()}
+				/>,
+				[emailToNoteDescription(), emailToNoteLabel(), accountEmailLabel(), accountTypeLabel(), accountInformationLabel()],
```

Comment:
This way of adding search text for custom components may need to be refactored at some point (though not in this pull request :) ).

One option could be to create a custom `<SearchableText>...</SearchableText>` component that sends search text to the settings screen with a custom [React context and provider](https://react.dev/reference/react/createContext).

Reply:
For now let's put only `emailToNoteLabel()` and `emailToNoteDescription()` in there